Compose/Simulators/README.md

```diff
@@ -0,0 +1,113 @@
+# TS Simulators
+A docker compose file (in yaml format) that invokes the T&S simulators required to run integration scripts.
+
+## Usage
+
+Clone the ts_Dockefiles repository:
+```sh
+ $ git clone -b feature/couger01/dm-18753 https://github.com/lsst-ts/tsDockerfiles.git
```

Comment:
Fix link to `ts_Dockerfiles` repo... should be https://github.com/lsst-ts/ts_Dockerfiles.git not https://github.com/lsst-ts/tsDockerfiles.git